pkg/unittest/default_cluster.go

```diff
@@ -114,3 +114,44 @@ func DefaultCluster() capiv1alpha2.Cluster {
 
 	return cluster
 }
+
+func DefaultClusterEmptyOrganization() *capiv1alpha2.Cluster {
```

Comment:
Created additional cases for organization label with empty value and without organization label.